### ISSUE TITLE
Function inoperative

### ISSUE THREAD

**Issue by DevMagnolia** [PseudoKnight/CHNaughty#1]
mc version: 1.12.2
commandhelper: 3.3.2#3378
chNaughty: 3.9.0

set_entity_rotation() <- This function does not work
pitch and yaw do not change the direction entity looks.

**Comment by PseudoKnight** [comment]
So, probably due to client interpolation of head look, some entities will not turn in small increments unless teleported. For example, a creeper didn't visually turn until it was at least 160 degrees from its initial yaw. Some entities, like armor stands and boats, can turn in small increments. 

Pitch always seems to work for me, though. Which entity did you test that with?

So the idea behind this function was that you couldn't turn an entity while it was riding another entity. In Bukkit, if you teleport an entity it dismounts it. I made this as an alternative to teleporting. When teleporting it always updates the rotation, so if the entity is not mounted, use set_entity_loc() instead.

I did more research and will continue to look at ways of improving this, but I think it's working as best it can.
